### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by michael-schnell**
Initital commit, inspired by quarkus

### CODE REVIEW

File: src/main/resources/io/smallrye/coderules/eclipse-format.xml [hunk 1]
```diff
@@ -0,0 +1,348 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="16">
+<profile kind="CodeFormatterProfile" name="smallrye" version="16">
```
Comment (kenfinnigan) [inline src/main/resources/io/smallrye/coderules/eclipse-format.xml]: Could you change the name to "SmallRye"

File: src/main/resources/io/smallrye/coderules/eclipse-format.xml [hunk 2]
```diff
@@ -0,0 +1,348 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (kenfinnigan) [inline src/main/resources/io/smallrye/coderules/eclipse-format.xml:1]: It's hard to tell as the list isn't ordered. Is this exactly the same as what Quarkus currently has or were there changes?

**Review by kenfinnigan — CHANGES_REQUESTED** [review]

File: src/main/resources/io/smallrye/coderules/eclipse-format.xml (hunk 2 above)
Comment (michael-schnell) [reply src/main/resources/io/smallrye/coderules/eclipse-format.xml:1]: I think the only change I made was the line length. It was changed from 120 to 140 chars. With normal screen size this should be fine.

**Comment by michael-schnell** [comment]
OK, name is changed and content is exactly  like Quarkus now.

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**